doc/ReleaseNotes.md

```diff
@@ -8,6 +8,7 @@ The release versions are NuGet releases.
 * check if BaseUri is absolute (see [PR #738](https://github.com/vvvv/SVG/pull/738))
 
 ### Fixes
+* fixed the parsing of significant whitespace (see [#744](https://github.com/vvvv/SVG/issues/744) and [PR #745](https://github.com/vvvv/SVG/pull/745))
```

Comment:
By convention, the list seems to be in ascending order.
Please add it below the list.